### DISCUSSION TITLE
Python 2.7.5

### DISCUSSION THREAD

**Discussion by igorrivin** [facebookresearch/faiss#2232] [Q&A]
This is an embarrassing question, but it would be very helpful for me to have a python 2.7.5 wrapper for FAISS. I am a complete novice in SWIG, so any suggestions or reasons why this might not be possible (hopefully there are not any) would be much appreciated.

**Comment by mausch** [comment]
Given that [official support for Python 2 was dropped on 1/1/2020](https://www.python.org/doc/sunset-python-2/) I think your options are either [pay someone to do it](https://www.activestate.com/products/python/python-2-7/) or do it yourself.
